build-system/tasks/storybook/README.md

```diff
@@ -0,0 +1,221 @@
+# Storybook
+
+## Style Recommendations
+
+### Do not use Knobs
```

Comment:
This first bit about knobs feels more like it should be an announcement to active contributors than a permanent part of the repo.

Reply:
Fair enough! I'll be notifying maintainers and adding a lint rule (#35991)